tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -535,7 +535,6 @@ def test_create_large_diff_snapshot(test_microvm_with_api):
     vm = test_microvm_with_api
```

Comment:
hmm I think this is empty because I cherry picked this commit into another series.